hack/jenkins/linux_integration_tests_kvm.sh

```diff
@@ -36,4 +36,9 @@ EXTRA_TEST_ARGS="-gvisor"
 mkdir -p cron && gsutil -qm rsync "gs://minikube-builds/${MINIKUBE_LOCATION}/cron" cron || echo "FAILED TO GET CRON FILES"
 sudo install cron/cleanup_and_reboot_Linux.sh /etc/cron.hourly/cleanup_and_reboot || echo "FAILED TO INSTALL CLEANUP"
 
+sudo apt-get update
+sudo apt-get -y install qemu-system libvirt-clients libvirt-daemon-system
+sudo adduser jenkins libvirt
```

Comment:
we probabl need || true if the user was allready added, I think it will give error

Reply:
fixed